lib/date_time_parser.ex

```diff
@@ -72,12 +72,12 @@ defmodule DateTimeParser do
     {:ok, DateTime.from_naive!(~N[2018-01-01T15:24:00Z], "Etc/UTC")}
     # or ~U[2018-01-01T15:24:00Z] in Elixir 1.9.0+
 
-    iex> DateTimeParser.parse_datetime(~s|"Dec 1, 2018 7:39:53 AM PST"|, to_utc: true)
-    {:ok, DateTime.from_naive!(~N[2018-12-01T14:39:53Z], "Etc/UTC")}
+    iex> DateTimeParser.parse_datetime(~s|"Mar 28, 2018 7:39:53 AM PDT"|, to_utc: true)
```

Comment:
Was this change to have `PDT` and `PST`? That seems valuable. I sort of also like the idea of having two of the same dates, one with `to_utc: true` and one with `to_utc: false`, but not sure if that provides more value.

Reply:
Daylight Time is from Mar10 to Nov3, so I changed the date here to be within PDT so the doctest will pass. There's another one that's in PST.

I'll add another `to_utc: false` test to mirror this.

Reply:
Daylight Time is from Mar10 to Nov3, so I changed the date here to be within PDT so the doctest will pass. There's another one that's in PST.

I'll add another to_utc: false test to mirror this.